### ISSUE TITLE
marks all unread emails as read

### ISSUE THREAD

**Issue by GoogleCodeExporter** [AkarshSatija/gmail-backup-com#1]
```
What steps will reproduce the problem?
1. Run gmail-backup.com
2. Goto mail.google.com
3. All emails in inbox are marked as read

What is the expected output? What do you see instead?
Same as before. All unread messages should stay unread. Instead all messages 
are marked as read.

What version of the product are you using? On what operating system?
revision 712 (2010-09-29)

Please provide any additional information below.


```

Original issue reported on code.google.com by `Peter.Po...@gmail.com` on 31 Jan 2011 at 6:46

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 20110307
```

Original comment by `honza.svec` on 25 Mar 2011 at 6:51
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Not fixed
```

Original comment by `turl...@gmail.com` on 9 Mar 2013 at 3:46
